lib/cocoapods-binary-cache/pod-rome/xcodebuild_raw.rb

```diff
@@ -18,15 +23,18 @@ def self.xcodebuild(options)
       targets.each { |target| cmd << "-target" << target }
       cmd << "-configuration" << options[:configuration]
       cmd << "-sdk" << sdk
-      unless platform.nil?
-        cmd << Fourflusher::SimControl.new.destination(:oldest, platform, options[:deployment_target])
+      if DESTINATION_OF_SDK.key?(sdk)
+        cmd << "-destination" << DESTINATION_OF_SDK[sdk]
+      else
+        cmd << Fourflusher::SimControl.new.destination(:oldest, platform, options[:deployment_target]) unless platform.nil?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/grab/cocoapods-binary-cache/blob/8afcbe7e9f825998a53c9c8e9114f9a1ed22543c/lib/cocoapods-binary-cache/pod-rome/xcodebuild_raw.rb#L29"></span>Layout/LineLength: Line is too long. [123/120]</td>
    </tr>
  </tbody>
</table>